iphone/Classes/TiFilesystemFileProxy.m

```diff
@@ -406,11 +406,17 @@ - (id)extension:(id)args
   return [path pathExtension];
 }
 
-- (id)getParent:(id)args
+- (NSString *)getParent:(id)args
 {
+  DEPRECATED_REPLACED(@"Filesystem.File.getParent()", @"7.0.0", @"Filesystem.File.parent");
   return [path stringByDeletingLastPathComponent];
 }
 
+- (TiFilesystemFileProxy *)parent
+{
+  return [[[TiFilesystemFileProxy alloc] initWithFile:[path stringByDeletingLastPathComponent]] autorelease];
```

Comment:
@sgtcoolguy Sorry to bug you on this, but to confirm:

1. Deprecate `getParent()` for parity with other platforms
2. Introduce `parent` property that works the same across all platforms
3. In 8.0.0+, remove the manual `getParent` accessor, so `getParent()` will use the computed getter that matches all platforms without a deprecation.